src/services/info/notice/notice-detail/NoticeDetailPage.vue

```diff
@@ -186,12 +169,46 @@ export default {
         /* Api */
         const getNoticePostData = async () => {
             try {
-                state.noticePostData = await SpaceConnector.client.board.post.get({
+                const results = await SpaceConnector.client.board.post.get({
                     board_id: props.boardId,
                     post_id: props.postId,
                 });
+                state.noticePostData = results;
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                state.noticePostData = {};
+            }
+        };
+        const getNextPostData = async (createdAt) => {
+            try {
+                const nextPostApiQueryHelper = new ApiQueryHelper()
+                    .setPage(1, 1)
+                    .setSort('created_at', false)
+                    .setFilters([{ k: 'created_at', v: createdAt, o: '>' }]);
```

Comment:
```suggestion
                    .setFilters([{ k: 'created_at', v: createdAt, o: '>t' }]);
```
datetime operator를 써야하지 않나요!? 잘 동작하나요??

Reply:
아 이거 나중에 말씀드려야지하고 잊을 뻔했네여.
datetime_gte랑 datetime_gt랑 같은 값이 나옵니다.
그래서 t를 제거하고 진행하니 잘 되더라구여..